bundles/org.openhab.binding.mybmw/src/main/java/org/openhab/binding/mybmw/internal/handler/MyBMWProxy.java

```diff
@@ -322,12 +323,12 @@ public synchronized boolean updateToken() {
             /*
              * Step 2) Calculate values for base parameters
              */
-            String verfifierBytes = Converter.getRandomString(64);
+            String verfifierBytes = StringUtils.getRandomAlphabetic(64).toLowerCase();
```

Comment:
The other sleepy comment aside, I think this would be slightly "more random" and also perform slightly better:
```suggestion
            String verfifierBytes = StringUtils.getRandomString(64, "abcdefghijklmnopqrstuvxyz");
```

WDYT?

Of course "abcdefghijklmnopqrstuvxyz" could then be extracted to a constant for being used on both places.

Reply:
There is no difference in randomness. 
Offcourse your suggestion is a good way of handling the case, the current implementation is also good. 
To me this is ‘the same but different’

Reply:
Okay, this was not obvious to me, but ChatGPT supports you. 😄

> The key factor in randomness is the random number generator (e.g., `SecureRandom`) used to select characters from the character set. As long as the random number generator is of high quality and evenly distributes the character selections across the character set, you will achieve the same level of randomness regardless of whether you generate a mixed-case string and then convert it to lowercase or directly generate a lowercase-only string. The lowercase conversion doesn't introduce any bias into the randomness of character selection.

And the performance difference is negligible, so let's merge your version.